### DISCUSSION TITLE
Is there a server-side cache available for Gatsby Functions?

### DISCUSSION THREAD

**Discussion by Vacilando** [gatsbyjs/gatsby#31971] [Help]
We can use Build Caching (https://www.gatsbyjs.com/docs/build-caching/) at build time, meaning heavy lifting jobs does not have to be performed at every build.

What kind of caching can we use with Gatsby Functions? 
For example, if we do heavy image transformations in a Gatsby function, is there a way to cache the transformed images for a given length of time, so that it does not have to happen at every request?

**Comment by KyleAMathews** [comment]
You can control caching with cache headers — see https://gatsby.canny.io/gatsby-cloud/p/add-support-for-headers-in-functions and https://github.com/gatsbyjs/gatsby/discussions/31842
